set/db.go

```diff
@@ -87,11 +87,37 @@ const (
 	workerPoolSizeFiles = 16
 )
 
+// DBRO is the read-only component of the DB struct.
+type DBRO struct {
+	db *bolt.DB
+
+	ch codec.Handle
+}
+
+// NewRO returns a *DBRO that can be used to query a set database. Provide
+// the path to the database file.
+//
+// Returns an error if path exists but can't be opened, or if it doesn't exist
+// and can't be created.
```

Comment:
Can't be created? Doesn't make sense for a RO method. Does dbOpenMode need to be different?